tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,8 @@
+export const checkLetters = (arr) => {
+  const lowerCaseArr = arr.map((cur) => cur.toLowerCase());
+  const secondElArr = lowerCaseArr[1].split('').map(
+      (cur) => lowerCaseArr[0].includes(cur) ? true : false);
```

Comment:
1.  is function "includes" return boolean?  do you need " ? true : false" ? Maybe also you don't need a arrow function  (cur) => lowerCaseArr[0].includes (cur) ? true : false) and instead of map(this lowerCaseArr[0].includes) should work